src/main/java/org/jabref/gui/groups/GroupTreeViewModel.java

```diff
@@ -623,7 +623,7 @@ public boolean canAddGroupsIn(GroupNodeViewModel groupnode) {
         } else if (group instanceof SearchGroup) {
             return true;
         } else if (group instanceof AutomaticKeywordGroup) {
-            return false;
+            return true;
```

Comment:
`true` allows sub groups of automatic keyword groups. This is not wished in the issue.

Reply:
ok